app/src/main/java/org/oppia/app/recyclerview/BindableAdapter.kt

```diff
@@ -7,6 +7,7 @@ import androidx.databinding.ViewDataBinding
 import androidx.recyclerview.widget.RecyclerView
 import kotlin.reflect.KClass
 
+
```

Comment:
Please remove. There should just be one empty line between imports and file members.

Reply:
Done